tests/islands_test.ts

```diff
@@ -478,3 +474,46 @@ Deno.test({
   sanitizeOps: false,
   sanitizeResources: false,
 });
+
+Deno.test({
+  name: "render nested islands with server children conditionally",
+
+  async fn(_t) {
```

Comment:
```suggestion
  async fn() {
```
Same with other test.